jwql/website/apps/jwql/bokeh_containers.py

```diff
@@ -62,29 +69,29 @@ def add_limit_boxes(fig, yellow=None, red=None):
         greater than the high value of yellow.
     """
     if yellow is not None:
-        green = BoxAnnotation(bottom=yellow_limits[0], top=yellow_limits[1], fill_color='chartreuse', fill_alpha=0.2)
```

Comment:
It still blows my mind that these errors made it into production. Somehow the yellow and red boxes are being added to plots at the moment.